### PR TITLE
Support passing a function to "clientSideParams" prop.

### PR DESCRIPTION
**Pull request by Melatonin64**
- Support passing a function to "clientSideParams" prop.
- Upgrade to latest @ekolabs/eko-js-sdk.
- Bump version.

### CODE REVIEW

**Comment by OpherV** [comment]
@Melatonin64 Something seems to be off as this fails the visual tests...

**Comment by Melatonin64** [comment]
@OpherV yes - would love help understanding why.
Would appreciate it if you could take a look - thanks!

**Comment by Melatonin64** [comment]
@OpherV @DivyaMahadevan got this to pass chromatic tests.
Ended up making some changes to the stories in chromatic mode so that screenshots will be more consistent.
Would appreciate if you can review - thanks!

File: src/stories/EkoVideo.stories.js [hunk 1]
```diff
@@ -85,12 +86,52 @@ NoAutoplay.args = {
...
+                // Disable ekoshell background color when paused
+                deliveryobject: JSON.stringify({
+                    environment: {
+                        playerOptionsOverrides: {
+                            plugins: {
+                                ekoshell: {
+                                    addons: {
+                                        projectmetadata: {
+                                            background: 'rgba(0,0,0,0)'
+                                        }
```
Comment (DivyaMahadevan) [inline src/stories/EkoVideo.stories.js:118]: @Melatonin64 the subtitles tests don't need this?

**Review by DivyaMahadevan — APPROVED** [review]
minor comment, but otherwise lgtm!

**Review by OpherV — APPROVED** [review]
Looks great! Thanks. I'm happy that the visual tests caught on the fact the default delivery API changed + the pause side effects were a little different

File: src/stories/EkoVideo.stories.js (hunk 1 above)
Comment (Melatonin64) [reply src/stories/EkoVideo.stories.js:118]: Not currently, as they run on embed API v1.0 (older delivery).

**Pull request merged**